charts/karpenter/templates/controller/rbac.yaml

```diff
@@ -74,7 +74,7 @@ rules:
   - patch
   - get
   - list
-  - patch
```

Comment:
This seems wrong if you're patching below

Reply:
We already have patch on #L74 so I changed this to update so we've always got both. I can just remove this line